src/pages/Login/Input/Input.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+
+import React from 'react';
+import '../Input/Input.scss';
+
+const Input = props => {
```

Comment:
```suggestion
const Input = ({isAlertPop, placeholder, inputRef}) => {
```
인자자리에서 바로 구조분해할당 할 수 있습니다!